x/perpetual/keeper/mtp_health.go

```diff
@@ -0,0 +1,53 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ammtypes "github.com/elys-network/elys/x/amm/types"
+	"github.com/elys-network/elys/x/perpetual/types"
+)
+
+// GetMTPHealth Health = custody / liabilities
+// It's responsibility of outer function to update mtp.BorrowInterestUnpaidLiability using UpdateMTPBorrowInterestUnpaidLiability
```

Comment:
so when doing a query, are we making sure latest health is returned ? (considering accrued, funding rate and interest rate)

Reply:
Yes. Need to call UpdateMTPBorrowInterestUnpaidLiability, it updates the value (doesn't set in KV store)
Earlier calling this in GetMTPHealth was causing to calculate interest amount multiple times (2-3 times) thus giving incorrect values

Reply:
got it, thanks